installer/charts/values.yaml.tpl

```diff
@@ -6,9 +6,10 @@
 {{- $gitops := required "GitOps settings" .Installer.Features.openShiftGitOps -}}
 {{- $pipelines := required "Pipelines settings" .Installer.Features.openShiftPipelines -}}
 {{- $quay := required "Quay settings" .Installer.Features.redHatQuay -}}
+{{- $quayEnabled := not (lookup "v1" "Secret" .Installer.Namespace "rhtap-nexus-integration") -}}
 {{- $rhdh := required "RHDH settings" .Installer.Features.redHatDeveloperHub -}}
 {{- $ingressDomain := required "OpenShift ingress domain" .OpenShift.Ingress.Domain -}}
-{{- $minIOOperatorEnabled := or $tpa.Enabled $quay.Enabled -}}
+{{- $minIOOperatorEnabled := or $tpa.Enabled $quayEnabled -}}
```

Comment:
This change looks wrong. Why do we need to install minIO if TPA is disabled and an external Nexus instance is used?

Reply:
Let's figure out the logic in the other thread, and then see how it impacts this.